### DISCUSSION TITLE
Tasmotized Sonoff Bridge R2 and Intertechno Equipment

### DISCUSSION THREAD

**Discussion by LupeChristoph** [arendst/Tasmota#15807] [Q&A] [answered]
Hello!

Forgive me if I'm infringing on conventions etc, I'm new both to Tasmota and in this forum. I searched extensively for a guide to control an Intertechno IT-1500 my new Sonoff bridge, but I'm unable to teach it with the ITLS-16 remote. Of course the remote is fine switching the IT-1500 directly, but I want to integrate that and two Intertechno dimmers in openHAB.

I tried `rfkey1 2` but the bridge does not receive anything from the remote. I found the tip to use `RfRaw AAB155` or `RfRaw 177` but that didn't help.

Can anybody please help me with instructions of a pointer to some helpful web page?

**Comment by sfromis** [comment, marked as answer]
With the Sonoff RF R2 bridge, you cannot flash Portisch, which provided more options on the prior generation, and therefore you cannot use raw mode.

**Reply by LupeChristoph** [reply]
Sorry, but I have no idea how the unavailability of raw mode affects me.

**Reply by Jason2866** [reply]
raw mode is only available when the custom firmware "Portisch" is flashed on the rf ic. Since the new Bridge has a other incompatible rf ic flashing is not possible of Portisch. Hence no raw mode.

**Reply by LupeChristoph** [reply]
Great. I found this out in the meantime myself. Except I still have no idea why I would need this to interface with the <expletive>Intertechno stuff</expletive>.

OK, I'll except I need this for reasons you know but expect me to find out myself. I'll send the Sonoff bridge back instead.

**Reply by Jason2866** [reply]
The standard rf firmware can recognize only a few rf protocols. Intertechno is NOT supported.
